packages/buttons/src/elements/ButtonGroup.js

```diff
@@ -93,7 +93,7 @@ export default class ButtonGroup extends ControlledComponent {
   };
 
   render() {
-    const { children, ...otherProps } = this.props; // eslint-disable-line no-unused-vars
+    const { children, onStateChange, ...otherProps } = this.props; // eslint-disable-line no-unused-vars
```

Comment:
What's this prop restructured for I can't see it used below?

Reply:
styled-components has a new props white-list that they allow. Unless we destructure the `onStateChange` value out there are warnings.

Reply:
Ah good to know, thanks.